packages/types/src/primitive/StorageKey.ts

```diff
@@ -69,7 +71,15 @@ export default class StorageKey extends Bytes {
       const [fn, ...arg]: [StorageFunction, ...Array<any>] = value as any;
 
       assert(isFunction(fn), 'Expected function input for key construction');
-
+      if (Array.isArray(arg[0])) {
+        const key = fn(...arg);
+        const u8a = new U8a(key);
+        return {
+          key: u8a,
```

Comment:
put them in one line like `key: new U8a(fn(...arg))`

Reply:
Yup. Okay